File: Dart/1044.dart
import 'dart:io';

void main() {
  var input = stdin.readLineSync();
  int a = int.parse(input.split((' '))[0]);
  int b = int.parse(input.split((' '))[1]);

  if (a % b == 0 || b % a == 0) {
    print('Sao Multiplos');
  } else {
    print('Nao sao Multiplos');
  }
}

File: C++/1044.cpp
#include <iostream>
using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;
    if (b % a == 0 || a % b == 0)
    {
        cout << "Sao Multiplos" << endl;
    }
    else
    {
        cout << "Nao sao Multiplos" << endl;
    }
    return 0;
}